app/utils/githubHelpers.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios'
+
+var id = "YOUR_CLIENT_ID";
+var sec = "YOUR_SECRET_ID";
+var param = `?client_id=${id}&client_secret=${sec}`;
+
+function getUserInfo (username = 'bartekkustra') {
+  return axios.get(`https://api.github.com/users/${username + param}`);
+}
+
+function getRepos (username = 'bartekkustra') {
```

Comment:
should there be a default user here? I believe when you provide no user it shows an error, and if you wanted to default, would probably do that at the caller and not in the function signature.

If a username is required for this function, should probably throw an error so you can catch it before shipping